tests/Service/SiteServiceTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Tests\Service;
+
+use App\Entity\Site;
+use App\Service\SiteService;
+use Doctrine\ORM\EntityManagerInterface;
+
+class SiteServiceTest extends ServiceTestCase
+{
+    protected $service;
+    protected $id;
+
+    public function testCaborConsentDisplay(): void
+    {
+        $this->id = uniqid();
+        $site = 'hpo-site-test' . $this->id;
+        $this->login('test@example.com', [$site]);
+        $this->createSite();
+        $this->service = static::$container->get(SiteService::class);
+        $this->service->switchSite($site . '@' . self::GROUP_DOMAIN);
+        self::assertFalse($this->service->displayCaborConsent());
+    }
+
+    public function createSite(): void
```

Comment:
Since this is only used inside other public test methods, can you make this private?